actions/installer/README.md

```diff
@@ -0,0 +1,29 @@
+# slsa-verifier setup GitHub Action
+
+This action installs the SLSA verifier and adds it to your PATH.
+
+For more information about `slsa-verifier`, refer to [its documentation](https://github.com/slsa-framework/slsa-verifier#verification-of-provenance).
+
+For more information about SLSA in general, see [https://slsa.dev](https://slsa.dev).
+
+## Usage
+
+To install a specific version of `slsa-verifier`, use:
+
+```yaml
+uses: slsa-framework/slsa-verifier-installer@v1.3.0
+```
+
+Note: Pinning this action to a release tag or commit SHA will pin you to the latest version of `slsa-verifier` at the time of release.
+
+### Optional Inputs
```

Comment:
there's no option, so lets remove this as well.

Reply:
Done